.env

```
@@ -1,20 +1,35 @@
-# For test end ponint which depends on the login
+# ═══════════════════════════════════════════════════════════
+#  Authentication & Security
+# ═══════════════════════════════════════════════════════════
+NEXTAUTH_URL="http://localhost:3000"
+NEXTAUTH_SECRET="FDZaz8bV6y8V9KTiJqjTxjs5Npvvm39hi+qmQZqK8fw="
+
+# Test Access Token (for development only - expires periodically)
 ACCESS_TOKEN="eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjoiNjdjMDYzOWQ4MzZlZThiZTcwNjE5MGE5Iiwicm9sZSI6ImFkbWluIiwiaWF0IjoxNzYwMTk5MzQ5fQ.E61wk2cgsIEC0IihKE8vVS999r0OWh88WIE3e94F_6o"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)